docs/actions/bazarliste.yaml

```
@@ -192,21 +209,25 @@ actions:
     label: Cartographie
     description: Les fiches seront sous forme de marqueur.
     hint: Le champ géolocalisation doit être présent dans votre formulaire
+    width: 35%
     properties:
       template:
         value: map
       provider:
         label: Fond de carte
         required: true
         type: list
-        default: OpenStreetMap.Mapnik
+        default: Stadia.OSMBright
```

```suggestion
        OpenStreetMap.Mapnik
```
Je n'aurais pas changé ce paramètre car sinon ça change le églage par défaut de `{{bazarcarto}}`.
Je préfère avant la fusion de la PR, remettre le paramètre d'avant (quitte à laisser un commentaire + TODO pout valider après concertation le changement de la valeur par défaut)

---

Ok